.github/workflows/integration-image-factory-cron.yaml

```diff
@@ -84,21 +84,6 @@ jobs:
         if: github.event_name == 'schedule'
         run: |
           make talosctl-cni-bundle
-      - name: iso
```

Comment:
this intentional?

Reply:
yes, I need to test this, but image factory cron uses only image factory assets, so why would it use an ISO?

Reply:
ahh okay, just wanted to be sure